File: typescript/src/useCase/SellItemsRequest.ts
import SellItemRequest from "./SellItemRequest";

class SellItemsRequest {
  private requests: SellItemRequest[];

  public setRequests(requests: SellItemRequest[]): void {
      this.requests = requests;
  }

  public getRequests(): SellItemRequest[] {
    return this.requests;
  }
}

export default SellItemsRequest;

File: csharp/TellDontAsk/UseCase/SellItemsRequest.cs
using System.Collections.Generic;

namespace TellDontAsk.UseCase
{
    public class SellItemsRequest
    {
        private List<SellItemRequest> requests;

        public void setRequests(List<SellItemRequest> requests)
        {
            this.requests = requests;
        }

        public List<SellItemRequest> getRequests()
        {
            return requests;
        }
    }
}
